pil-splitter.c

```diff
@@ -10,7 +10,11 @@
 #include <stdbool.h>
 #include <fcntl.h>
 #include <unistd.h>
-#include <elf.h>
+#if defined(__APPLE__)
+        #include "include/elf.h"
+#else
+        #include <elf.h>
+#endif
```

Comment:
Everything else aside, it seems cleaner to add the local `include` folder to the search path from the makefile perhaps? And then rename the folder to indicate that it's a specific fallback for when the target doesn't provide `elf.h`.